src/main/java/com/powsybl/openloadflow/sensi/SensitivityFactorReader.java

```diff
@@ -13,11 +13,29 @@
  */
 public interface SensitivityFactorReader {
 
+    interface ContingencyContext { }
```

Comment:
Coudn't we replace this marker interface by something like:
```
enum ContingencyContextType {
   ALL, NONE, SPECIFIC
}
class ContingencyContext {
   private final String contingencyId;
   private final ContingencyContextType contextType;
}
```
Not the prettiest design either I know, but at least this avoids casting and instance of.